specs/operations/20260819_h1_official_contract_worklog.md

```
@@ -0,0 +1,136 @@
+  `tests/fixtures/official_layout/*`
+
+公式で確認した H1 の値域:
+
+| 項目 | 公式値 |
+| --- | --- |
+| データ区分 | `0` 削除 / `2` 前日最終売上 / `4` 最終 / `5` 月曜最終 / `9` レース中止 |
+| 発売フラグ 1〜7 | `0` 発売なし / `1` 発売前取消 / `3` 発売後取消 / `7` 発売あり |
+| 複勝着払キー | `0` 複勝発売なし / `2` 2着まで / `3` 3着まで |
+| 返還馬番/枠番/同枠 | 位置ごとの `0`/`1`、28 / 8 / 8 桁、初期値 `0` |
+| 票数 | 11 バイト、単位百円、ALL0 は発売前取消し・発売票数なし |
+| 人気順 | 数字（単勝・複勝・枠連 2 桁、馬連・ワイド・馬単・3連複 3 桁）、
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)